torch/fx/passes/runtime_assert.py

```diff
@@ -14,11 +14,13 @@
 from torch.fx._utils import get_node_context, lazy_format_graph_code
 from torch.fx.experimental.sym_node import SymNode
 from torch.fx.graph_module import GraphModule
+from torch.fx._compatibility import compatibility
 
 log = logging.getLogger(__name__)
 graph_code_log = torch._logging.getArtifactLogger(__name__, "graph_code")
 
 
+@compatibility(is_backward_compatible=True)
 def get_example_value(node: fx.Node) -> Optional[str]:
```

Comment:
We can land this as is to unblock but this function should be private

Reply:
I need to update anyway since, as usual, I forgot to run lintrunner before submitting...